test/e2e/infinispan/upgrade_operand_test.go

```diff
@@ -322,18 +322,17 @@ func TestSpecImageUpdate(t *testing.T) {
 	// release. We can then manually specify spec.image using the FQN of the latest image to simulate a user specifying
 	// custom images
 	replicas := 1
-	versionManager := tutils.VersionManager()
-	operand := versionManager.Operands[len(versionManager.Operands)-2]
+	imageOperand, versionOperand := specImageOperands()
```

Comment:
Maybe we can make these variable names more informative, how about "firstOperand, secondOperand"? I don't think "image" and "version" have much meaning in the context of this test.

The comment also needs updating.